pandas/pandas_interoperability

```diff
@@ -0,0 +1,27 @@
+import Orange
+import pandas as pd
+import numpy as np
+
+
+def construct_domain(df):
+    columns = OrderedDict(df.dtypes)
+
+    def create_variable(col):
+        if col[1].__str__().startswith('float'):
```

Comment:
Hello, this is an error I have. I don't know if it is related.
Simply reading a dataset from a text file. I am running Orange in Windows
10 64 bits python 3.5


[image: Imágenes integradas 1]

2015-12-11 19:42 GMT+01:00 kernc <notifications@github.com>:

> In pandas/pandas_interoperability
> <https://github.com/biolab/orange3/pull/796#discussion_r47388601>:
>
> > @@ -0,0 +1,27 @@
> > +import Orange
> > +import pandas as pd
> > +import numpy as np
> > +
> > +
> > +def construct_domain(df):
> > +    columns = OrderedDict(df.dtypes)
> > +
> > +    def create_variable(col):
> > +        if col[1].__str__().startswith('float'):
>
> What is the code that reproduces this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biolab/orange3/pull/796/files#r47388601>.
>



-- 
/ .- .-.. .-.. / -.-- --- ..- / -. . . -.. / .. ... / .-.. --- ...- .
José Antonio Martín H. (PhD)           E-Mail: jamartinh@fdi.ucm.es
Computer Science Faculty               Phone: (+34) 91 3947650
Complutense University of Madrid       Fax: (+34) 91 3947527
C/ Prof. José García Santesmases,s/n   28040 Madrid, Spain
web: http://www.dacya.ucm.es/jam/
LinkedIn: http://www.linkedin.com/in/jamartinh (Let's connect)
.-.. --- ...- . / .. ... / .- .-.. .-.. / .-- . / -. . . -..